cohttp-async/bin/cohttp_curl_async.ml

```diff
@@ -31,7 +31,7 @@ let make_net_req uri meth' body () =
   show_headers (Cohttp.Response.headers res);
   body
   |> Body.to_pipe
-  |> Pipe.iter ~f:(fun b -> Caml.Pervasives.print_string b; return ())
+  |> Pipe.iter ~f:(fun b -> Stdlib.print_string b; return ())
```

Comment:
Doesn’t this also require to depend on `stdlib-shims` for `cohttp-async`?

Reply:
It does! Good catch, fixing now.